modules/reference/pages/systems/systems-list.adoc

```diff
@@ -134,7 +134,8 @@ The menu:Systems[Systems > Inactive Systems] page displays systems that have not
 image::systems_inactive.png[scaledwidth=80%]
 
 Checking in means that Zypper on {sle} or Red Hat Update Agent on {rhel} client systems connects to {productname} to see if there are any updates available or if any actions have been scheduled.
-If you see a message telling you that check-ins are not taking place, the client system is not successfully connecting to {productname}.
+In case of salt systems, there is a taskomatic job that checks on the minions by performing a test.ping on them.
```

Comment:
https://github.com/SUSE/doc-susemanager/pull/384

(I tried to  do one of those little inline PRs, but failed, somehow. *sigh*)

Reply:
It's called "Suggested Changes" and it's available in the GitHub Beta program.

https://github.blog/changelog/2018-10-16-suggested-changes/

Reply:
Incorporating @Loquacity's suggestion:

```suggestion
For Salt systems, a Taskomatic job checks on the minions periodically by pinging them when otherwise inactive.
If you see the message " System not checking in with {productname}" in a system's Detail page, then the system is not successfully connecting to {productname} (in the case of Salt minions, Taskomatic's ping mentioned above is not successful).
```

Reply:
Hrm, I can't see 'suggest changes' anywhere, I'll have to keep looking ;)